test/controllers/projects/samples/transfer_controller_test.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
```

Comment:
Can you update the name of this test to `TransfersControllerTest` and the file name to `transfers_controller_test.rb` to match the controller name

Reply:
I missed that, thanks. Please see 18eaf248b20cb128556b5907809f44cd7b2654fa.

Reply:
You're welcome and thank you for updating it!